general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -183,7 +183,7 @@ def create_job(
             about the job, but does contain the job id.
 
         Raises:
-            An SuperstaqException if the request fails.
+            An SuperstaqServerException if the request fails.
```

Comment:
nit
```suggestion
            gss.SuperstaqServerException: If the request fails.
```
(also outside the scope of this pr but we should really decide between `gss.SuperstaqServerException:` and `SuperstatqServerException:` for these)